spec/support/matchers/have_delivered_notification.rb

```diff
@@ -6,14 +6,24 @@
   match do |proc|
     raise ArgumentError, "have_delivered_notification only supports block expectations" unless proc.respond_to?(:call)
 
-    expected_attributes = {
-      type: notification_type,
+    expected_notification_attributes = {
+      type: "#{notification_type}::Notification",
       params: (hash_including(expected_params) if expected_params),
       recipient: expected_recipient,
     }.compact
 
-    expect(proc).to change { Notification.count }.by(1)
-    expect(Notification.last).to have_attributes(expected_attributes)
+    expected_event_attributes = {
+      type: notification_type,
+      params: (hash_including(expected_params) if expected_params),
+    }
+
+    expect(proc).to change { Noticed::Notification.count }.by(1)
+
+    notification = Noticed::Notification.last
+    event = Noticed::Event.find(notification.event_id)
+
+    expect(notification).to have_attributes(expected_notification_attributes)
+    expect(event).to have_attributes(expected_event_attributes)
```

Comment:
We now have two models/tables rather than just one and they store different attributes